### PR TITLE
Added support for gentoo

### PR DESCRIPTION
**Pull request by Rocco83**
just added gentoo things :)

### CODE REVIEW

**Comment by Rocco83** [comment]
Any comment on this PR?
Will be ever merged?

**Comment by dhoppe** [comment]
@Rocco83 Sorry, I missed this PR somehow. I do not have an Gentoo running, but I trust your expertise. Could you please do a rebase before I hit the merge button?

**Comment by Rocco83** [comment]
Of course!

File: CHANGELOG.md [hunk 1]
```diff
@@ -60,6 +66,7 @@
 - [Gem] Update list of required gems
 - [Git] Update list of ignored files/directories
 - [Beaker] Update box/box_url because of new point release
+>>>>>>> 6da429d9b2ef8f4dd5ca813cb49efebe659bebfe
```
Comment (dhoppe) [inline CHANGELOG.md]: @Rocco83 You missed that during the merge.

Comment (Rocco83) [inline CHANGELOG.md]: done, sorry for that

Comment (Rocco83) [inline CHANGELOG.md]: About hash, absolutely correct, i left it in push.
About several release missing, https://github.com/dhoppe/puppet-postfix/pull/5/files can you point out? very likely di have not understood what you mean.

**Comment by dhoppe** [comment]
@Rocco83 Never mind. Thx for your contribution / patience.

**Pull request merged**